Traceback (most recent call last):
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 188, in <module>
    main()
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 174, in main
    process_json_and_shapes(
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 157, in process_json_and_shapes
    save_response(json_path, question_id, response)
  File "/p/project1/hai_kg-rag-thesis/scripts/KG_Agent_MK2/call_llm_api.py", line 85, in save_response
    raise ValueError(f"❌ ERROR: Question ID {question_id} not found in JSON file.")
ValueError: ❌ ERROR: Question ID 0 not found in JSON file.
